AraCorrelator/RayTraceCorrelator.cxx

```diff
@@ -336,15 +336,94 @@ std::vector<TGraph> RayTraceCorrelator::GetCorrFunctions(
             for (int isamp=0; isamp<nPoints; isamp++){
                 tVals.push_back(xvals[isamp]);
                 vVals.push_back(yvals[isamp]);
-            }
+            }            
             TGraph grCorrOut(nPoints, &tVals[0], &vVals[0]);
-            
             corrFunctions.push_back(grCorrOut);
         }
     }
     return corrFunctions;
 }
 
+std::pair< 
+    std::vector< std::vector< std::vector< int > > >,
+    std::vector< std::vector< std::vector< double > > > > RayTraceCorrelator::GetArrivalDelays(std::map<int, std::vector<int > > pairs){
+
+    std::vector< std::vector< std::vector< double > > > delays;
+    std::vector< std::vector< std::vector< int > > > bins;
+
+    for(int solNum=0; solNum<2; solNum++){
+
+        std::vector< std::vector<double> > delays_this_sol;
+        std::vector< std::vector<int> > bins_this_sol;
+
+        for(auto iter = pairs.begin(); iter != pairs.end(); ++iter){
+            int pairNum = iter->first;
+            int ant1 = iter->second[0];
+            int ant2 = iter->second[1];
+
+            std::vector<double> this_delays;
+            std::vector<int> this_global_bins;
+
+            for(int phiBin=0; phiBin < this->numPhiBins_; phiBin++){
+                for(int thetaBin=0; thetaBin < this->numThetaBins_; thetaBin++){
+                    
+                    int globalBin = (phiBin + 1) + (thetaBin + 1) * (this->numPhiBins_ + 2);
+
+                    if ((globalBin<0) or (globalBin > 65522)){
```

Comment:
I'm not sure if this makes sense? Ideally the max value wouldn't be hard coded in case we change resolution and if you do it based on the resolution then it's impossible given the loop I think?

Reply:
That is a debugging remnant, sorry. Removing.